actors/common/src/main/java/org/sunbird/learner/actors/OrganisationManagementActor.java

```diff
@@ -478,17 +478,17 @@ private void createOrg(Request actorMessage) {
 
   private void createOrgExternalIdRecord(String channel, String externalId, String orgId) {
     Map<String, Object> orgExtIdRequest = new HashMap<String, Object>();
-    orgExtIdRequest.put(JsonKey.PROVIDER, channel);
-    orgExtIdRequest.put(JsonKey.EXTERNAL_ID, externalId);
+    orgExtIdRequest.put(JsonKey.PROVIDER, channel.toLowerCase());
```

Comment:
in this method there is no null check for channel and externalId, we should not assume caller will do that check, so add this check. Incase of null or empty , either logs it as info or throw error , if it's  mandatory.

Reply:
StringUtils.lowerCase - would handle null situation as well.